src/promptflow-tools/promptflow/tools/open_source_llm.py

```diff
@@ -618,15 +704,93 @@ def format_response_payload(self, output: bytes) -> str:
             print(error_message, file=sys.stderr)
             raise OpenSourceLLMOnlineEndpointError(message=error_message)
 
+class ServerlessLlamaContentFormatter(ContentFormatterBase):
+    """Content formatter for LLaMa"""
+
+    def __init__(self, api: API, chat_history: Optional[str] = ""):
+        super().__init__()
+        self.api = api
+        self.chat_history = chat_history
+        self.model_id = "llama-2-7b-hf"
+
+    @staticmethod
+    def parse_chat(chat_str: str) -> List[Dict[str, str]]:
```

Comment:
This is duplicate code, can we refactor it.

Reply:
Maybe it can go to ContentFormatterBase itself, because i dont think this part needs to change based on each model.